recipes/z5py/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.5.3" %}
+# {% set checksum = "c644f3457b31be2fa0ce8fce15536e2eaeab160d" %}
+
+package:
+  name: z5py
+  version: {{ version }}
+
+source:
+    url: https://github.com/constantinpape/z5/archive/{{ version }}.tar.gz
+    # sha256: {{ checksum }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - boost 1.67.0
+    - xtensor-python
+    - bzip2
+    - c-blosc
+    - xz
+    - zlib
+  run:
+    - python
+    - boost 1.67.0
+    - numpy # use {{ pin_compatible('numpy') }} instead ?
```

Comment:
Is `numpy` needed at build time (e.g. it includes headers from/links to `numpy`)? For example [`vigra` does]( https://github.com/conda-forge/vigra-feedstock/blob/5a6034233f4b1a9520c1eac331e9fcb8cb34b221/recipe/meta.yaml#L40 ) hence it has [the `pin_compatible` line]( https://github.com/conda-forge/vigra-feedstock/blob/5a6034233f4b1a9520c1eac331e9fcb8cb34b221/recipe/meta.yaml#L51 ). If we don't need `numpy` at build time, we shouldn't need the `pin_compatible` line.

Reply:
Yes, numpy is needed at build time. It is pulled in as a dependency of `xtensor-python`.
Maybe  it would be a good idea to list it as explicit dependency and then pin the version?